sdk/src/main/java/io/opentelemetry/sdk/resources/Resource.java

```diff
@@ -45,6 +47,26 @@
   private static final String ERROR_MESSAGE_INVALID_VALUE =
       " should be a ASCII string with a length not exceed " + MAX_LENGTH + " characters.";
   private static final Resource EMPTY = create(Attributes.empty());
+  private static final Resource TELEMETRY_SDK =
+      create(
+          Attributes.newBuilder()
+              .setAttribute("telemetry.sdk.name", "opentelemetry")
+              .setAttribute("telemetry.sdk.language", "java")
+              .setAttribute("telemetry.sdk.version", readVersion())
+              .build());
+
+  @Nullable
+  private static String readVersion() {
+    Properties properties = new Properties();
+    try {
+      properties.load(
+          new FileInputStream(Resource.class.getResource("version.properties").getFile()));
```

Comment:
I think `FileInputStream` only works when loading from the file system, e.g., `IntelliJ` but not from a JAR. Instead

`Resource.class.getResourceAsAstream("version.properties")` can handle both